packages/botonic-plugin-contentful/src/contentful/cms-contentful.ts

```diff
@@ -235,10 +235,13 @@ export class Contentful implements cms.CMS {
     )
   }
 
-  async content(id: string, context = DEFAULT_CONTEXT): Promise<Content> {
-    const REFERENCES_TO_INCLUDE = 6
+  async content(
+    id: string,
+    context = DEFAULT_CONTEXT,
+    referencesToInclude = 6
```

Comment:
Sorry to be so picky, but I think that it's better to define a `DEFAULT_REFERENCES` constant and set that constant to `6` instead of setting the default value to `6` in all methods.

Reply:
applied!